ApiBundle/Facade/BlockCollectionFacade.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace OpenOrchestra\ApiBundle\Facade;
+
+use JMS\Serializer\Annotation as Serializer;
+
+/**
+ * Class BlockCollectionFacade
+ */
+class BlockCollectionFacade extends AbstractFacade
+{
+    /**
+     * @Serializer\Type("string")
+     */
+    public $collectionName = 'blocks';
```

Comment:
Si je l'enlève, j'obtiens un joli :

Property OpenOrchestra\ApiBundle\Facade\BlockCollectionFacade::$collectionName does not exist